### ISSUE TITLE
Fixed sizes for table

### ISSUE THREAD

**Issue by PhantomKunai** [frappe/print_designer#317]
Is there perhaps a way to maintain fixed row height for a table?

I am attempting to print 1 row per page (this is basically labels for items). In this case, if I match the row height and table container height, there will be 1 row per page.

**Comment by maharshivpatel** [comment]
@PhantomKunai what you are asking for is page break at row level ?

**Comment by PhantomKunai** [comment]
@maharshivpatel 
You could say that. Ideally the goal is to be able to print labels for a list of items in a child table, and each item appears on one page. Right now I am unable to 
1. break after a row (or)
2. set the row height such that it spans the page height and automatically page breaks as it works now

**Comment by maharshivpatel** [comment]
@PhantomKunai if you don't mind me asking why not design 1 label and print using list view ?

**Comment by PhantomKunai** [comment]
Here I designed a label. And in here I am trying to print a child table as this will contain specific information pertaining to that item received ( example, inspection seal, date received and so on )

![image](https://github.com/frappe/print_designer/assets/12585952/bd730c8f-aeb6-48b2-9a86-aa3b56da39e5)

It seems counter productive to create a separate doc for each label and print from the list view.

We achieved this previously by using jinja to parse through tables and setting minimum and maximum dimensions for elements to fit each in one page.

**Comment by PhantomKunai** [comment]
Ideally, each number should be on one page. Generating 6 pages.

**Comment by maharshivpatel** [comment]
maybe, option to access per row data and interface for printing each row per document can work ?

**Comment by PhantomKunai** [comment]
Yes that would seem like a solution for this. However, I think really it must be possible to build some html components using jinja. As I understand it now, Jinja is mainly used only to pre-process data in Print Designer.

Perhaps some blocks which act as containers and placeholders to simply render some html using jinja within its position on the page.

**Comment by maharshivpatel** [comment]
yeah, already on my mind 😅. but will take some time.
